sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3822,6 +3822,14 @@ object SQLConf {
       .version("4.1.0")
       .fallbackConf(SHUFFLE_DEPENDENCY_FILE_CLEANUP_ENABLED)
 
+  val THRIFT_SHUFFLE_DEPENDENCY_FILE_CLEANUP_ENABLED =
+    buildConf("spark.sql.thrift.shuffleDependency.fileCleanup.enabled")
```

Comment:
looking at other thriftserver confs like `spark.sql.thriftServer.interruptOnCancel`, I think we should name it `spark.sql.thriftServer.shuffleDependency.fileCleanup.enabled`